CRITICAL - 2025-06-28 16:36:52 --> CodeIgniter\Exceptions\CriticalError: The required PHP extension "mysqli" is not loaded. Install and enable it to use "MySQLi" driver.
[Method: GET, Route: user/all]
in SYSTEMPATH/Database/Database.php on line 182.
 1 SYSTEMPATH/Database/Database.php(59): CodeIgniter\Database\Database->checkDbExtension('MySQLi')
 2 SYSTEMPATH/Database/Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 3 SYSTEMPATH/Model.php(155): CodeIgniter\Database\Config::connect('default')
 4 APPPATH/Models/UserModel.php(14): CodeIgniter\Model->__construct()
 5 APPPATH/Controllers/UserController.php(16): App\Models\UserModel->__construct()
 6 SYSTEMPATH/CodeIgniter.php(904): App\Controllers\UserController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH/CodeIgniter.php(498): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 16:42:39 --> CodeIgniter\Exceptions\CriticalError: The required PHP extension "mysqli" is not loaded. Install and enable it to use "MySQLi" driver.
[Method: GET, Route: user/all]
in SYSTEMPATH/Database/Database.php on line 182.
 1 SYSTEMPATH/Database/Database.php(59): CodeIgniter\Database\Database->checkDbExtension('MySQLi')
 2 SYSTEMPATH/Database/Config.php(82): CodeIgniter\Database\Database->load([...], 'default')
 3 SYSTEMPATH/Model.php(155): CodeIgniter\Database\Config::connect('default')
 4 APPPATH/Models/UserModel.php(14): CodeIgniter\Model->__construct()
 5 APPPATH/Controllers/UserController.php(16): App\Models\UserModel->__construct()
 6 SYSTEMPATH/CodeIgniter.php(904): App\Controllers\UserController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
 7 SYSTEMPATH/CodeIgniter.php(498): CodeIgniter\CodeIgniter->createController()
 8 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-06-28 16:52:28 --> Error connecting to the database: mysqli_sql_exception: No such file or directory in /var/www/html/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('localhost', 'ci4user', Object(SensitiveParameterValue), 'ci4', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#6 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#7 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#8 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No such file or directory in /var/www/html/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#5 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#6 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
ERROR - 2025-06-28 16:55:47 --> Error connecting to the database: mysqli_sql_exception: No such file or directory in /var/www/html/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('localhost', 'ci4user', Object(SensitiveParameterValue), 'ci4db', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#6 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#7 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#8 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No such file or directory in /var/www/html/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#5 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#6 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
ERROR - 2025-06-28 16:59:21 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /var/www/html/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('127.0.0.1', 'ci4user', Object(SensitiveParameterValue), 'ci4db', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#6 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#7 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#8 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /var/www/html/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#5 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#6 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
ERROR - 2025-06-28 16:59:49 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /var/www/html/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('127.0.0.1', 'ci4user', Object(SensitiveParameterValue), 'ci4db', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#6 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#7 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#8 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /var/www/html/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#5 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#6 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
ERROR - 2025-06-28 17:02:18 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /var/www/html/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('127.0.0.1', 'ci4user', Object(SensitiveParameterValue), 'ci4db', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#6 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#7 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#8 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /var/www/html/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#5 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#6 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
ERROR - 2025-06-28 17:06:34 --> Error connecting to the database: mysqli_sql_exception: Connection refused in /var/www/html/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('db', 'ci4user', Object(SensitiveParameterValue), 'ci4db', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#6 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#7 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#8 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Connection refused in /var/www/html/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#5 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#6 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
ERROR - 2025-06-28 17:13:12 --> Error connecting to the database: mysqli_sql_exception: No such file or directory in /var/www/html/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('localhost', 'ci4user', Object(SensitiveParameterValue), 'ci4db', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#6 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#7 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#8 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No such file or directory in /var/www/html/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#5 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#6 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
ERROR - 2025-06-28 17:15:28 --> Error connecting to the database: mysqli_sql_exception: No such file or directory in /var/www/html/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('localhost', 'ci4user', Object(SensitiveParameterValue), 'ci4db', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#6 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#7 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#8 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No such file or directory in /var/www/html/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#5 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#6 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
ERROR - 2025-06-28 17:15:32 --> Error connecting to the database: mysqli_sql_exception: No such file or directory in /var/www/html/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('localhost', 'ci4user', Object(SensitiveParameterValue), 'ci4db', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#6 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#7 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#8 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No such file or directory in /var/www/html/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#5 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#6 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
ERROR - 2025-06-28 17:15:46 --> Error connecting to the database: mysqli_sql_exception: No such file or directory in /var/www/html/system/Database/MySQLi/Connection.php:201
Stack trace:
#0 /var/www/html/system/Database/MySQLi/Connection.php(201): mysqli->real_connect('localhost', 'ci4user', Object(SensitiveParameterValue), 'ci4db', 3306, '', 0)
#1 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#6 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#7 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#8 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No such file or directory in /var/www/html/system/Database/MySQLi/Connection.php:246
Stack trace:
#0 /var/www/html/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/system/Database/BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/system/Database/BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/app/Models/UserModel.php(24): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/app/Controllers/UserController.php(21): App\Models\UserModel->getUsers(Array, Array)
#5 /var/www/html/system/CodeIgniter.php(933): App\Controllers\UserController->getAllClients()
#6 /var/www/html/system/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
#7 /var/www/html/system/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /var/www/html/system/Boot.php(334): CodeIgniter\CodeIgniter->run()
#9 /var/www/html/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#10 /var/www/html/public/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#11 {main}
CRITICAL - 2025-06-28 17:52:02 --> ParseError: syntax error, unexpected token ")", expecting identifier or variable or "{" or "$"
[Method: GET, Route: user/a31067fd-5445-11f0-a0bf-0242c0a830]
in APPPATH/Controllers/UserController.php on line 35.
 1 SYSTEMPATH/Autoloader/Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('/var/www/html/app/Controllers/UserController.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\UserController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\UserController')
 4 SYSTEMPATH/CodeIgniter.php(885): class_exists('\\App\\Controllers\\UserController', true)
 5 SYSTEMPATH/CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 17:53:57 --> TypeError: App\Controllers\BaseController::failNotFound(): Argument #1 ($description) must be of type string, stdClass given, called in /var/www/html/app/Controllers/UserController.php on line 36
[Method: GET, Route: user/a31067fd-5445-11f0-a0bf-0242c0a830]
in SYSTEMPATH/API/ResponseTrait.php on line 222.
 1 APPPATH/Controllers/UserController.php(36): App\Controllers\BaseController->failNotFound(Object(stdClass))
 2 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\UserController->getClient('a31067fd-5445-11f0-a0bf-0242c0a830')
 3 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 4 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 17:55:21 --> ParseError: syntax error, unexpected variable "$this"
[Method: GET, Route: user/a31067fd-5445-11f0-a0bf-0242c0a830]
in APPPATH/Controllers/UserController.php on line 37.
 1 SYSTEMPATH/Autoloader/Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('/var/www/html/app/Controllers/UserController.php')
 2 SYSTEMPATH/Autoloader/Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\UserController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\UserController')
 4 SYSTEMPATH/CodeIgniter.php(885): class_exists('\\App\\Controllers\\UserController', true)
 5 SYSTEMPATH/CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:00:09 --> Error: Call to undefined function App\Controllers\passwordValidationRule()
[Method: POST, Route: user/]
in APPPATH/Controllers/UserController.php on line 51.
 1 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\UserController->createUser()
 2 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 3 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:12:41 --> CodeIgniter\Filters\Exceptions\FilterException: "authFilter" filter must have a matching alias defined.
[Method: GET, Route: user/all]
in SYSTEMPATH/Filters/Filters.php on line 590.
 1 SYSTEMPATH/Filters/Filters.php(590): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('authFilter')
 2 SYSTEMPATH/Filters/Filters.php(641): CodeIgniter\Filters\Filters->enableFilter('authFilter', 'before')
 3 SYSTEMPATH/CodeIgniter.php(467): CodeIgniter\Filters\Filters->enableFilters([...], 'before')
 4 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:13:07 --> CodeIgniter\Filters\Exceptions\FilterException: "authFilter" filter must have a matching alias defined.
[Method: GET, Route: user/all]
in SYSTEMPATH/Filters/Filters.php on line 590.
 1 SYSTEMPATH/Filters/Filters.php(590): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('authFilter')
 2 SYSTEMPATH/Filters/Filters.php(641): CodeIgniter\Filters\Filters->enableFilter('authFilter', 'before')
 3 SYSTEMPATH/CodeIgniter.php(467): CodeIgniter\Filters\Filters->enableFilters([...], 'before')
 4 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:14:52 --> CodeIgniter\Filters\Exceptions\FilterException: "JwtAuthFilter" filter must have a matching alias defined.
[Method: GET, Route: user/all]
in SYSTEMPATH/Filters/Filters.php on line 590.
 1 SYSTEMPATH/Filters/Filters.php(590): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('JwtAuthFilter')
 2 SYSTEMPATH/Filters/Filters.php(641): CodeIgniter\Filters\Filters->enableFilter('JwtAuthFilter', 'before')
 3 SYSTEMPATH/CodeIgniter.php(467): CodeIgniter\Filters\Filters->enableFilters([...], 'before')
 4 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:18:12 --> CodeIgniter\Filters\Exceptions\FilterException: "JwtAuthFilter" filter must have a matching alias defined.
[Method: GET, Route: user/all]
in SYSTEMPATH/Filters/Filters.php on line 590.
 1 SYSTEMPATH/Filters/Filters.php(590): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('JwtAuthFilter')
 2 SYSTEMPATH/Filters/Filters.php(641): CodeIgniter\Filters\Filters->enableFilter('JwtAuthFilter', 'before')
 3 SYSTEMPATH/CodeIgniter.php(467): CodeIgniter\Filters\Filters->enableFilters([...], 'before')
 4 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:20:01 --> CodeIgniter\Filters\Exceptions\FilterException: "JwtAuthFilter" filter must have a matching alias defined.
[Method: GET, Route: user/all]
in SYSTEMPATH/Filters/Filters.php on line 590.
 1 SYSTEMPATH/Filters/Filters.php(590): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('JwtAuthFilter')
 2 SYSTEMPATH/Filters/Filters.php(641): CodeIgniter\Filters\Filters->enableFilter('JwtAuthFilter', 'before')
 3 SYSTEMPATH/CodeIgniter.php(467): CodeIgniter\Filters\Filters->enableFilters([...], 'before')
 4 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:20:26 --> CodeIgniter\Filters\Exceptions\FilterException: "JwtAuthFilter" filter must have a matching alias defined.
[Method: GET, Route: user/all]
in SYSTEMPATH/Filters/Filters.php on line 590.
 1 SYSTEMPATH/Filters/Filters.php(590): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('JwtAuthFilter')
 2 SYSTEMPATH/Filters/Filters.php(641): CodeIgniter\Filters\Filters->enableFilter('JwtAuthFilter', 'before')
 3 SYSTEMPATH/CodeIgniter.php(467): CodeIgniter\Filters\Filters->enableFilters([...], 'before')
 4 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:21:08 --> CodeIgniter\Filters\Exceptions\FilterException: "JwtAuthFilter" filter must have a matching alias defined.
[Method: GET, Route: user/all]
in SYSTEMPATH/Filters/Filters.php on line 590.
 1 SYSTEMPATH/Filters/Filters.php(590): CodeIgniter\Filters\Exceptions\FilterException::forNoAlias('JwtAuthFilter')
 2 SYSTEMPATH/Filters/Filters.php(641): CodeIgniter\Filters\Filters->enableFilter('JwtAuthFilter', 'before')
 3 SYSTEMPATH/CodeIgniter.php(467): CodeIgniter\Filters\Filters->enableFilters([...], 'before')
 4 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:27:42 --> CodeIgniter\Database\Exceptions\DatabaseException: You must set the database table to be used with your query.
[Method: POST, Route: auth/login]
in SYSTEMPATH/Database/BaseConnection.php on line 941.
 1 SYSTEMPATH/Model.php(714): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH/Model.php(920): CodeIgniter\Model->builder()
 3 APPPATH/Controllers/AuthController.php(23): CodeIgniter\Model->__call('where', [...])
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->login()
 5 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 6 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:28:12 --> CodeIgniter\Database\Exceptions\DatabaseException: You must set the database table to be used with your query.
[Method: POST, Route: auth/login]
in SYSTEMPATH/Database/BaseConnection.php on line 941.
 1 SYSTEMPATH/Model.php(714): CodeIgniter\Database\BaseConnection->table(null)
 2 SYSTEMPATH/Model.php(920): CodeIgniter\Model->builder()
 3 APPPATH/Controllers/AuthController.php(23): CodeIgniter\Model->__call('where', [...])
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->login()
 5 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 6 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:35:05 --> Error: Call to undefined method stdClass::first()
[Method: POST, Route: auth/login]
in APPPATH/Controllers/AuthController.php on line 23.
 1 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->login()
 2 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:35:27 --> Error: Cannot use object of type stdClass as array
[Method: POST, Route: auth/login]
in APPPATH/Controllers/AuthController.php on line 25.
 1 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->login()
 2 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:41:34 --> Error: Cannot use object of type stdClass as array
[Method: POST, Route: auth/login]
in APPPATH/Controllers/AuthController.php on line 24.
 1 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->login()
 2 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-06-28 21:42:16 --> Error: Cannot use object of type stdClass as array
[Method: POST, Route: auth/login]
in APPPATH/Controllers/AuthController.php on line 25.
 1 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\AuthController->login()
 2 SYSTEMPATH/CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH/CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH/Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH/index.php(63): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
